README.md

```diff
@@ -1,5 +1,43 @@
 # Node.js Internationalization
 
+## Contributing
+
+Help us get the documentations translated to your native language and reach more people's heart.
+**Visit [Crowdin/project/NodeJs](https://crowdin.com/project/nodejs)** and help translating docs
+
+### Languages being Translated
+
+- Chinese Simplified - [zh-CN](https://crowdin.com/project/nodejs/zh-CN)
```

Comment:
I think it would be cleaner to leave the locales out and just use the language names:

```
[Chinese Simplified](https://crowdin.com/project/nodejs/zh-CN)
```

I would also suggest ordering this list alphabetically.